Modules/Core/Support/CountryHelper.php

```
@@ -0,0 +1,46 @@
+     *
+     * @param $cldr
+     * @param $countrycode
+     *
+     * @return mixed
+     */
+    public static function get_country_name($cldr, $countrycode)
+    {
+        $countries = get_country_list($cldr);
+    
+        return $countries[$countrycode] ?? $countrycode;
+    }
```

PHPMD (2.15.0)

The method get_country_name is not named in camelCase. (undefined)

(CamelCaseMethodName)